### ISSUE TITLE
Add unit tests

### ISSUE THREAD

**Issue by ufechner7** [tecosaur/About.jl#38]
Unit tests are useful to ensure that existing functionality does not brake with pull requests or on different machine architectures or new Julia versions.

**Comment by tecosaur** [comment]
Indeed, however it is a little difficult to effectively test a display-oriented package like this.
